BUILD_INSTRUCTIONS.markdown

```diff
@@ -16,11 +16,17 @@ token after creating an account here:
 
 https://api.portal.trustedservices.intel.com/EPID-attestation
 
+Clone the Veracruz repository:
+
+```
+git clone git@github.com:veracruz-project/veracruz.git
+export VERACRUZ_ROOT=$PWD/veracruz
+```
 Once you have a local copy of the Veracruz source and your token:
 
 ```
-cd SGXRustOPTEEDevelDocker
-make VERACRUZ_ROOT=<path to your veracruz directory> IAS_TOKEN=<your Intel Attestation Service token>
+cd veracruz-docker-image
+make IAS_TOKEN=<your Intel Attestation Service token>
```

Comment:
Don't we also need to set VERACRUZ_ROOT?

Reply:
VERACRUZ_ROOT is set in a previous step after the repo clone : I could add some more detail explaining that ?

Reply:
Doesn't the make command itself need to hae VERACRUZ_ROOT=...,, or does an environment variable work? (I can't recall)

Reply:
@dreemkiller: `make` mirrors environment variables from the environment within which it is invoked as implicit make variables.

Reply:
cool.